templates/robotics/ros-esm.html

```diff
@@ -91,40 +94,33 @@ <h3 class="p-muted-heading">
   </div>
 </section>
 
-<section class="p-strip ">
-  <div class="row">
-    <div class="col-6">
-      <h2>
-        Benefits of ROS ESM
-      </h2>
-    </div>
-  </div>
-  <div class="row u-equal-height u-vertically-center">
-    <div class="col-8">
-      <h3>
-        Maintain your ROS environment
-      </h3>
-      <p>
-        Just like the rest of your software, ROS needs regular maintenance as projects scale. The more robots you have, the more tedious, expensive and time-consuming the process gets.
-      </p>
-      <p>
-        ROS ESM provides you with continuous maintenance of your ROS environment through CVE and critical bug fixes.
-      </p>
-    </div>
-    <div class="col-4 u-align--center u-vertically-center u-hide--small">
+<section class="p-strip">
+  <div class="u-fixed-width">
+    <div class="u-align-text--center">
```

Comment:
Would it be better if there is a little bit more space between the digram and the texts? It might not be necessary but what do you think? :) 
```suggestion
    <div class="u-align-text--center u-sv3">
```